### DISCUSSION TITLE
How to install beta 3  v.2 on Mac?

### DISCUSSION THREAD

**Discussion by SamuelMiller** [Zettlr/Zettlr#4182] [General]
The export function for Mermaid charts in the nightly builds doesn't appear to be working on Mac, Catalina OS. How do I install the official beta 3 v.2 version with proper pandoc support as announced here https://github.com/Zettlr/Zettlr/releases?

**Comment by nathanlesage** [comment]
macOS Catalina is unfortunately no longer supported. The minimum required macOS version is 11.3.

**Reply by SamuelMiller** [reply]
Thanks.
